mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule QRCode.MixProject do
       app: :qr_code,
       dialyzer: dialyzer_base() |> dialyzer_ptl(System.get_env("SEMAPHORE_CACHE_DIR")),
       version: "0.1.0",
-      elixir: "~> 1.7",
+      elixir: "~> 1.8",
```

Comment:
We use some incompatibity from *1.8*?

Reply:
At lower elixir version (<1.8) is not functional `floor` function in guards.